redis/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM redis:4.0.14-alpine3.10
+FROM redis:4.0.14-alpine3.11
```

Comment:
We should update our Airflow images too -- they are based of 3.10 -- and annoyingly have `alpine-3.10` in the name.

Perhaps for those a rebuild against latest alpine3.10 is enough.